flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -380,11 +393,7 @@ class SentryFlutterOptions extends SentryOptions {
 class _SentryFlutterExperimentalOptions {
   /// Replay recording configuration.
   final replay = SentryReplayOptions();
-}
 
-/// Callback being executed in [ScreenshotEventProcessor], deciding if a
-/// screenshot should be recorded and attached.
-typedef BeforeScreenshotCallback = FutureOr<bool> Function(
-  SentryEvent event, {
-  Hint? hint,
-});
+  /// Privacy configuration for masking sensitive data in the Screenshot and Session Replay.
+  SentryPrivacyOptions? privacy;
```

Comment:
I've made suggestion how we can make this more user-friendly not having to initialize the field) in another PR on top of this one: #2382